korman/ui/ui_toolbox.py

```diff
@@ -50,6 +50,13 @@ def draw(self, context):
         col.operator("texture.plasma_toggle_environment_maps", icon="IMAGE_RGB", text="Enable All EnvMaps").enable = True
         col.operator("texture.plasma_toggle_environment_maps", icon="IMAGE_RGB_ALPHA", text="Disable All EnvMaps").enable = False
 
+        # Double Sided Operators
+        col.label("Double Sided:")
+        col.operator("mesh.plasma_toggle_double_sided", icon="MESH_DATA", text="Disable All").enable = False
+        mesh_list = [i.data for i in bpy.context.selected_objects if i.type == "MESH"]
+        all_double_sided = all((mesh.show_double_sided for mesh in mesh_list))
```

Comment:
You could simplify this by merging these two lines:
```python
all_double_sided = all((mesh.show_double_sided for i in bpy.context.selected_objects if i.type == "MESH"))
```

Reply:
Shouldn't that be `i.show_double_sided`?

Reply:
Oh, yes. Whoops.

Reply:
I see where you were going with that and devised this:
`all_double_sided = all((mesh.show_double_sided for mesh in [i.data for i in bpy.context.selected_objects if i.type == "MESH"]))`

Reply:
I was hoping to get rid of the list part `[i.data for i in bpy.context.selected_objects if i.type == "MESH"]` because it stores all the meshes then processes them. I see what I have wrong in the code I gave you previously...

```python
all_double_sided = all((i.data.show_double_sided for i in bpy.context.selected_objects if i.type == "MESH"))
```